diff --git a/src/Autofac/Features/OpenGenerics/OpenGenericRegistrationSource.cs b/src/Autofac/Features/OpenGenerics/OpenGenericRegistrationSource.cs
index 6d842273..b7fc22f1 100644
--- a/src/Autofac/Features/OpenGenerics/OpenGenericRegistrationSource.cs
+++ b/src/Autofac/Features/OpenGenerics/OpenGenericRegistrationSource.cs
@@ -17,4 +17,5 @@ internal class OpenGenericRegistrationSource : IRegistrationSource
     private readonly RegistrationData _registrationData;
     private readonly IResolvePipelineBuilder _existingPipelineBuilder;
+    [System.Reflection.Obfuscation(Feature = "Renaming")] // Is accessed via reflection.
     private readonly ReflectionActivatorData _activatorData;
 
