diff --git a/.config/dicts/project-words.txt b/.config/dicts/project-words.txt
index 58f0fec..e796a2a 100644
--- a/.config/dicts/project-words.txt
+++ b/.config/dicts/project-words.txt
@@ -6,3 +6,4 @@ varint
 unflushed
 Reentrancy
-sharpfuzz
\ No newline at end of file
+sharpfuzz
+Vilens
diff --git a/Test.ps1 b/Test.ps1
index effcc8a..93c0def 100644
--- a/Test.ps1
+++ b/Test.ps1
@@ -12,8 +12,7 @@ $ErrorActionPreference = 'Stop'
 $PSNativeCommandUseErrorActionPreference = $true
 
-cspell lint $PSScriptRoot
 
 # Run regular tests using MTP
-dotnet test --disable-logo --configuration $Configuration
+dotnet test --disable-logo --configuration $Configuration --solution "$PSScriptRoot/tests/UnitTests.slnf"
 
 # Publish AOT tests
